geomesa-z3/src/main/scala/org/locationtech/geomesa/zorder/sfcurve/Z2.scala

```diff
@@ -0,0 +1,90 @@
+/***********************************************************************
```

Comment:
So the plan is to inherit sfcurve into gm rather than publishing 2.13 artifacts?

Reply:
yeah, we're only using a couple classes, and the maintainers are AWOL. Jim did finally publish a 2.13 jar for me, but it took months. since the project hasn't seen any activity in ~6 years iirc, i think it's ok to just pull it in.